assets/js/base/context/providers/cart-checkout/payment-methods/use-payment-method-registration.ts

```diff
@@ -75,7 +76,14 @@ const usePaymentMethodRegistration = (
 			cart,
 			cartTotals,
 			cartNeedsShipping,
-			billingData,
+			get billingData() {
+				/* eslint no-console: ["error", { allow: ["warn"] }] */
+				console.warn(
```

Comment:
Could we use https://developer.wordpress.org/block-editor/reference-guides/packages/packages-deprecated/ here?

Reply:
@opr I changed the section before to:

```ts
get billingData() {
  // prettier-ignore
  deprecated(
    'billingData',
    {
      alternative: 'billingAddress',
      plugin: 'woocommerce-gutenberg-products-block',
      link:
        'https://github.com/woocommerce/woocommerce-blocks/pull/6369',
    }
  );
  return this.billingAddress;
},
```

Is that what you had in mind?